[[preparation_structure]]
== Dataset preparation and structure

In this section you will find detailed information and best practices on how to prepare and structure datasets for uploading and processing. If you still feel uncertain, you may wish to take a look at some of the <<example_data>>.

NOTE: Illustrations (screenshots) may be slighlty different from what you see in the online tool, as the MDT is still being developed and improved.

TIP: You may want to read sections of the general https://doi.org/10.35035/doc-vf1a-nr22[DNA publishing guide] for background information. Specifically: https://docs.gbif.org/publishing-dna-derived-data/en/#rationale[Rationale], https://docs.gbif.org/publishing-dna-derived-data/en/#environmental-dna-as-a-source-for-dna-derived-occurrence-data[Environmental DNA as a source...], https://docs.gbif.org/publishing-dna-derived-data/en/#dna-metabarcoding-sequence-derived-data[DNA metabarcodin data], https://docs.gbif.org/publishing-dna-derived-data/en/#introduction-to-biodiversity-publishing[Introduction to publishing], https://docs.gbif.org/publishing-dna-derived-data/en/#taxonomy-of-sequences[Taxonomy of sequences], https://docs.gbif.org/publishing-dna-derived-data/en/#taxonomy-of-sequences[Mapping metabarcoding data]

IMPORTANT: The general https://doi.org/10.35035/doc-vf1a-nr22[DNA publishing guide] contains a lot of advice and guidance on data preparation and mapping (incl required/recommended fields). the MDT will handle most of these things automatically (incl some fields). If you can fit your data into one of the provided OTU table templates here, the processing should be almost self-explanatory.

=== The OTU table as a familiar data format

The tool accepts data in a format familiar to the eDNA metabarcoding community - the OTU table.

The OTU table is a specialized version of a species/site matrix, which is a standard way to organize data in biodiversity and ecology studies, summarizing the presence and abundance of different organisms across various samples. In an OTU table each row represents a unique OTU, and each column represents a different sample or site, with the cell values indicating the count of sequencing reads of the OTUs in each sample. An OTU table may also have OTUs as columns and samples as rows – i.e. transposed compared to the above description.

The two dimensions (columns and rows) of the central OTU table is most often associated with two other tables:

* A table with metadata relating to the samples/sites – e.g. geographical position and sampling date, pH, salinity, habitat type, sample age, etc. – that allows the user to analyse how community compositional measures (betadiversity) correlate with geography, environment, etc. 
* A table relating to the OTUs – at least the sequence, but likely also inferred taxonomy, and maybe inferred ecological traits of these taxa – that allows the user to evaluate taxonomic or functional differences between samples.

NOTE:  It is commonly seen that an OTU table (with OTUs as rows) has the taxonomic information (sequence, kingdom, phylum, etc) added as extra columns. Also we see sample metadata (position, sampling date, soil ph, etc) added as extra columns to an OTU table (with samples as rows). We recommend to work with a clean OTU_table only containing read counts of OTUs in samples, and have the associated Sample and OTU metadate in separate tables linked by OTU IDs and sample IDs.

Most analysed datasets from DNA metabarcoding will be possible to fit into this three-table format.

the MDT uses the OTU table as a publishing model, and reshapes the data into a format that can be published to GBIF. 

the MDT accepts OTU table data in one of the provided dataset templates. Each dataset template contains the three tables mentioned above (as sheets in an Excel workbook or as separate tsv/csv files). To minimize redundant data entry, a fourth table can be used for metadata with values that are identical across the whole dataset. Sequences can also be provided in a separate txt file (as a fasta file.)

* *OTU_table*: The table (matrix) of sequence read counts of each OTU in each sample
* *Taxonomy*: a table of the sequences and potentially their taxonomy. Only rarely this table holds more fields, as relevant metadata (sequencing platform, primers) is similar across all samples/OTUs, and can be put into the *Study* table.
* *Samples*: a table of the metadata associated with the samples. This table is where all the metadata with different values for each sample is placed (geography, sampling date, etc.).
* *Study*: this (optional) fourth table contain only two columns (term, value). This table can be used for metadata (terms) with values that are the same for the whole dataset ("global values") – often that would be things like: target gene, primer information, sequencing platform, pipeline, extraction procedure, etc.
* *Seqs.fasta*: this (optional) file may contain the sequences of the OTUs instead of having them as a field in the *Taxonomy* table. 

.The OTU table. A minimalistic example of an *OTU_table* with 5 samples (Sample_A, Sample_B,..) as columns and 4 OTUs (OTU_001, OTU_002,..) as rows. Green boxes and arrow indicate the linking of the OTUs by their IDs to second table *Taxonomy* with taxonomic/sequence data data. Blue boxes and arrow indicate the the linking of samples by their IDs to the third table *Samples* with and sample metadata. The optional *Study* and *Seqs.fasta* are explained above.
image::img/OTU_table_all_filed.png[]

=== Preparing a dataset

==== Basic checks and preparations

* The data is DNA metabarcoding data (amplified and sequenced marker genes) and not something else (metagenomics, specimen barcoding, qPCR)
* You have access to the processed data as OTU table and associated table (and not just access to the raw un-analysed sequence data - e.g. fastq files)
* You are allowed to share the data in GBIF.
* You have the most essential metadata (e.g. location, date for the samples)
* You have an OTU table with number of sequence reads of each OTU in each sample
* You have identified which samples are not real samples (e.g. negative controls, positive controls, PCR blanks). These often have names hinting at their identity ("Neg", "Blank","Pos", "CTRL", ..), and often lack metadata (e.g. latitude/longitude).
** You have a plan for how to exclude these non-samples. The easiest approach is the remove them from the *Samples* sheet.
* You have the sequences of the OTUs/ASVs
* You have identified whether any OTUs are considered contaminations or non trustworthy detections ("orchids in the ocean" and "blue whales in the soil") to be excluded.
** You have a plan for how to exclude these contaminations. The easiest approach is the remove them from the *Taxonomy* sheet.

* You have some information that allows you to construct a dataset description
** You have information on the persons that should be associated as creators and contacts for the dataset. 

==== Select a template

Select one of the <<templates>> that is suitable for your data.

* If you prefer Excel workbooks, then use template 1 og 2.
* If you prefer tab (or comma) separated text files, go for template 3 or 4. 
* If you have sequences in a separate fasta file, then use template 2 or 4.

[[fit_data]]
==== Fit the data into the template

Now fit the data into the selected template and provide as many of the <<req_recom, required and recommended fields>> as possible.

TIP: The order (sorting) of rows and columns in the tables is not important. 

*OTU_table*

* Fit your data into an *OTU_table* with samples as columns and OTUs as row.
* Remove rows and columns that are not samples and OTUs to get a clean OTU/Sample matrix (i.e. remove rows or columns with sums/totals, etc.).
** If present, move columns with sample or taxomomy data to those separate tables.
* Use 0 (zero) in cells without a read count (not "NA", "-" or similar).
* Make sure that sample IDs (column headers) are unique, and prefarably do not contain a lot of non-standard characters. (NB: These sample IDs will automatically be used as term:dwc[eventID] by the MDT).
* Make sure that OTU IDs (row names) are unique, and prefarably do not contain a lot of non-standard characters.
* Leave the upper left cell empty.
+
NOTE: This table does not hold any DwC fields/terms

*Taxonomy*

* Fit your OTU (sequence/taxonomy) metadata into a *Taxonomy* table with OTUs as rows and associated fields as columns.
* First colums should be labelled `id` and contain OTU IDs referring to (identical to!) the row names (OTU IDs) in the *OTU_table*.
+
NOTE: Normally the *Taxonomy* table would not contain a lot of fields, as most fields that relate to this part is global (primers, sequencing platform, etc).
* The *Taxonomy* table would typically include:
** The sequence (as term:dwc[DNA_sequence]) unless these are provided in a separate fasta file.
** Taxonomy (inferred from comparing the sequences againat a reference database). Taxonomy can be given in several ways:
*** XXXX (how can we advice in a simple way about this)
+
NOTE: most metadata related to OTUs (sequencing platform, primers, etc) is identical for all samples and OTUs, and these can be placed in the fourth table *Study* with "global" values. 
* Use Darwin Core terms for your fields to minimize manual mapping in later steps.
+
TIP: known contaminants or other OTUs not wanted in GBIF.org can simply be removed from the *Taxonomy* table alone (and left untouched in the OTU table) 



*Samples*

* Fit your Sample metadata into a *Sample* table with Sample IDs as rows and associated data as columns.
* First colums should be `id` and contain Sample IDs referring to (identical to) the column names in the *OTU_table*. (NB: These IDs will automatically be used as term:dwc[eventID] by the MDT).
* This table is where you fit all the metadata related to the single samples if the values are different between samples, e.g. information on sampling locations, sampling dates, physical properties related to the sample (e.g. pH), and links to sample-associated data elsewhere – e.g. raw sequence data and biosample record in INSDC.
* Be sure to include as many of the required and recommended fields as possible (see below).
* In addition to the required/recommended fields, Occurrence Core (LINK) and dna-derived extension (link) has many further fields to select from.
+
TIP: Use *Study* table for fields/terms that does not differ between samples (i.e. terms with global values).
* Use Darwin Core terms for your fields to minimize manual mapping in later steps. 
+
TIP: known control samples or other samples not wanted in GBIF.org can simply be removed from the *Samples* table alone (and left untouched in the OTU table)

*Study* (optional)

The use of this table is optional. But as many metadata values often are applicable to the whole study (e.g. primers, sequencing platform, country, habitat type), we recommend to use this table, as opposed to having those fields in the *Samples* table with identical values for all samples.

* the table has two columns (_term_, _value_). Each row holds any term from Darwin Core (incl the dna-derived extension) in the _term_ field, and the corresponding _value_ contains the value relevant for this study (see minimal example below).
* Fit all metadata fields with global values (same value for all samples and/or OTUs) into this table.
* Be sure to include as many of the required and recommended fields as possible (see below).
* Use only Darwin Core terms for your fields. Manual mapping isnot possible for fields provided in this sheet.

image:img/study_table.png[]

*Seqs.fasta* (optional)

In some metabarcoding datasets, the sequences are placed in a separate https://en.wikipedia.org/wiki/FASTA_format[fasta file^]. This is also possible here. If this option is selected, then a few checks are good.

* Make sure you follow the classic fasta formatting for the file. A sequence begins with a greater-than character (">") followed by the OTU ID (exactly as they are given in the OTU table). The lines immediately following this header are the sequence representation (ACTG ...). The next ">" marks the beginning of the next sequence.
* OTU IDs (headers) should be the same as those in the OTU table, but with the ">" added in front.
* Some fasta files may have line breaks in the DNA sequence. the MDT allows this format. 

image:img/fasta_file.png[]

*Dataset descriptions, people and other metadata*

Before starting the data upload and processing in the MDT it is a good idea to prepare a dataset description, collect information on the people you need to associate with the data, etc.

You need to prepare:

* A dataset title. Choose something descriptive. Examples:
** XXX 
** XXX
** XXX
* A dataset description. XXXX[some advice on this]
* Persons that should be associated with the dataset. This could be the authors of an associated research paper, the laboratory personnel, the person preparing the dataset for GBIF publication, etc.
** Name
** Affiliation
** Address
** email
** ORCID (if available)
* All persons added will be listed as authors in the suggested dataset citation.
* Be sure to designate one of the persons (you?) as the contact person. This is the person that will be contacted if e.g. users find issues in the data. This person needs to be registered with email and ORCID.
* Associated ressources ???

NOTE: the MDT uses a  minimalistic web form for providing dataset metadata. This is intentional as we do not wish the demotivate users by confronting them with a web form with many options only marginally relevant for DNA metabarcoding data. If you chose to publish the processed data through an <<ipt>>, you will have the possibility of adding more/other dataset metadata there.

[[req_recom]]
=== Required and recommended fields

This section contains an overview of required and recommended fields to include in your dataset.

==== Quick reference on required and recommended fields

*Fields to be provided by user*

* Required: term:mixs[DNA_sequence], term:dwc[scientificName], term:dwc[eventDate], term:dwc[eventID] (Sample ID is used).

* Highly recommended: term:dwc[materialSampleID], term:dwc[recordedBy], term:dwc[decimalLatitude], term:dwc[decimalLongitude], term:mixs[target_gene], term:mixs[target_subfragment], term:mixs[pcr_primer_forward], term:mixs[pcr_primer_reverse], term:mixs[pcr_primer_name_forward], term:mixs[pcr_primer_name_reverse],term:mixs[pcr_primer_reference], term:mixs[seq_meth], term:mixs[otu_class_appr], term:mixs[otu_seq_comp_appr], term:mixs[otu_db], term:dwc[kingdom].

* Recommended: term:mixs[env_broad_scale], term:mixs[env_local_scale], term:mixs[env_medium], term:dwc[associatedSequences], term:mixs[lib_layout], term:mixs[sop], term:dwc[samplingProtocol], term:dwc[identificationRemarks], term:dwc[identificationReferences], term:dwc[phylum], term:dwc[class], term:dwc[order], term:dwc[family], term:dwc[genus].

*Fields automatically handled/filled by the MDT*

* term:dwc[basisOfRecord], term:dwc[occurrenceID], term:dwc[organismQuantity], term:dwc[organismQuantityType], term:dwc[sampleSizeValue], term:dwc[sampleSizeUnit], term:dwc[taxonID], (term:dwc[eventID] – the MDT automatically uses the proveded sample IDs).

==== Detailed reference on required and recommended fields

These tables are modified/specialized version of tables in the dna-publishing guide in the section https://docs.gbif.org/publishing-dna-derived-data/en/#mapping-metabarcoding-edna-and-barcoding-data[Mapping metabarcoding (eDNA) and barcoding data]. 

When using the MDT it is not important to know whether the fields you are using are from Occurrence Core or the dna-derived extension. But it is important to know in which table to put the fields and associated values. The *Placement* column explains where to provide each field. A lot of the required and recommended fields are automatically handled/calculated, and should not be provided by the user – *Placement* informs about this also.

NOTE: Many fields relating to taxonomy/sequences (e.g. primers, reference database) are most often possible to give as global values in the *Study* table. If you find that this is not the case, you may be dealing with a *mixed dataset* – i.e. with OTUs/sequences from more than one primer set (e.g. COI _and_ 16S sequences) from the same set of samples. We recommend to publish such datasets separately.

[[table-01]]
.Recommended fields for http://rs.gbif.org/core/dwc_occurrence_2020-04-15.xml[Occurrence core] for Metabarcoding data. This table is a modified version of table 2 in the https://docs.gbif.org/publishing-dna-derived-data/en/#mapping-metabarcoding-edna-and-barcoding-data[dna-publishing guide] specifically for using the MDT.
[cols="1,1,4,1,1",options="header"]
|===
| Field name
| Examples / explanation
| Description
| Required
| Placement

| term:dwc[basisOfRecord]
| _This field is is automatically set as "Material Sample" by the tool_
| The specific nature of the data record - a subtype of the http://rs.gbif.org/vocabulary/dwc/basis_of_record.xml[dcterms:type].
| Required
| _Automatic by tool_

| term:dwc[occurrenceID]
| _This field is automatilly contructed by the tool as "eventID:OTU_id"_
| A unique identifier for the occurrence, allowing the same occurrence to be recognized across dataset versions as well as through data downloads and use.
| Required
| _Automatic by tool_

| term:dwc[eventID]
| _This field is automatically set to the IDs of the Sample_
| An identifier for the set of information associated with an Event (something that occurs at a place and time).
| Highly recommended
| _Automatic by tool_

| term:dwc[eventDate]
| 2020-01-05
| Date when the event was recorded. Recommended best practice is to use a date that conforms to ISO 8601-1:2019. For more information, check https://dwc.tdwg.org/terms/#dwc:eventDate
| Required
| *Samples* (or *Study*)

| term:dwc[recordedBy]
| "Oliver P. Pearson \| Anita K. Pearson"
| A list (concatenated and separated) of names of people, groups, or organizations responsible for recording the original Occurrence. The recommended best practice is to separate the values with a vertical bar (' \| '). Including information about the observer improves the scientific reproducibility (https://doi.org/10.1093/database/baaa072[Groom et al. 2020^]).
| Highly recommended
| *Samples* or *Study*

| term:dwc[organismQuantity]
| _This field is automatically filled with the value from the corresponding cell in the uploaded OTU table_
| Number of reads of this OTU or ASV in the sample.
| Highly recommended
| _Automatic by tool_

| term:dwc[organismQuantityType]
| _This field is automatically filled with the value "DNA sequence reads"_
| Should always be “DNA sequence reads”
| Highly recommended
| _Automatic by tool_

| term:dwc[sampleSizeValue]
| _This field is automatically filled total number of reads in the sample as calculated by the tool automatically_
| Total number of reads in the sample. This is important since it allows calculating the relative abundance of each OTU or ASV within the sample.
| Highly recommended
| _Automatic by tool_

| term:dwc[sampleSizeUnit]
| DNA sequence reads
| _This field is automatically filled with the value “DNA sequence reads”_
| Highly recommended
| _Automatic by tool_

| term:dwc[materialSampleID]
| https://www.ncbi.nlm.nih.gov/biosample/15224856 +
 +
https://www.ebi.ac.uk/ena/browser/view/SAMEA3724543 +
 +
urn:uuid:a964805b-33c2-439a-beaa-6379ebbfcd03
| An identifier for the MaterialSample (as opposed to a particular digital record of the material sample). Use the biosample ID if one was obtained from a nucleotide archive. In the absence of a persistent global unique identifier, construct one from a combination of identifiers in the record that will most closely make the materialSampleID globally unique.
| Highly recommended
| *Samples*

| term:dwc[samplingProtocol]
| UV light trap
| The name of, reference to, or description of the method or protocol used during a sampling Event. https://dwc.tdwg.org/terms/#dwc:samplingProtocol
| Recommended
| *Study* (or *Samples*)

| term:dwc[associatedSequences]
| https://www.ebi.ac.uk/ena/browser/view/ERR1202046
| A list (concatenated and separated) of identifiers (publication, global unique identifier, URI). For most cases it woule be linking to archived raw metabarcoding read files in a public repository.
| Recommended
| *Samples*

| term:dwc[identificationRemarks]
| RDP annotation confidence (at lowest specified taxon): 0.96, against reference database: GTDB
| Specification of taxonomic identification process, ideally including data on applied algorithm and reference database, as well as on level of confidence in the resulting identification.
| Recommended
| *Study* (or *Taxonomy*)

| term:dwc[identificationReferences]
| https://www.ebi.ac.uk/metagenomics/pipelines/4.1 + 
 +
https://github.com/terrimporter/CO1Classifier
| A list (concatenated and separated) of references (publication, global unique identifier, URI) used in the Identification. Recommended best practice is to separate the values in a list with space vertical bar space ( \| ).
| Recommended
| *Study* (or *Taxonomy*)

| term:dwc[decimalLatitude]
| 60.545207
| The geographic latitude (in decimal degrees, using the spatial reference system given in geodeticDatum) of the geographic centre of a Location. Positive values are north of the Equator, negative values are south of it. Legal values lie between -90 and 90, inclusive.
| Highly recommended
| *Samples* (or *Study*)

| term:dwc[decimalLongitude]
| 24.174556
| The geographic longitude (in decimal degrees, using the spatial reference system given in geodeticDatum) of the geographic centre of a Location. Positive values are east of the Greenwich Meridian, negative values are west of it. Legal values lie between -180 and 180, inclusive.
| Highly recommended
| *Samples* (or *Study*)

// The [.break-all]#ASV:…# is to allow the identifier to be broken at any character, rather than stretching the text cell.
| term:dwc[taxonID]
| _This field is automatically filled with an MD5 hash of the sequence – e.g. [.break-all]#ASV:7bdb57487bee022ba30c03c3e7ca50e1#_
| For eDNA data, it is recommended to use an MD5 hash of the sequence and prepend it with “ASV:”. See also <<taxonomy-of-sequences>>.
| Highly recommended
| _Automatic by tool_

| term:dwc[scientificName]
| _Gadus morhua_ L. 1758, BOLD:ACF1143
| Scientific name of the closest known taxon (species or higher) or an OTU identifier from BOLD (BIN) or UNITE (SH)
| Required (filled with "Incertae sedis" if left blank)
| *Taxonomy*

| term:dwc[kingdom]
| Animalia
| Higher taxonomy
| Highly recommended
| *Taxonomy*

| term:dwc[phylum]
| Chordata
| Higher taxonomy
| Recommended
| *Taxonomy*

| term:dwc[class]
| Actinopterygii
| Higher taxonomy
| Recommended
| *Taxonomy*

| term:dwc[order]
| Gadiformes
| Higher taxonomy
| Recommended
| *Taxonomy*

| term:dwc[family]
| Gadidae
| Higher taxonomy
| Recommended
| *Taxonomy*

| term:dwc[genus]
| _Gadus_
| Higher taxonomy
| Recommended
| *Taxonomy*

|===

<<<

[[table-02]]
.Recommended fields from the DNA derived data extension (a selection) for metabarcoding data. This table is a modified version of table 3 in the https://docs.gbif.org/publishing-dna-derived-data/en/#mapping-metabarcoding-edna-and-barcoding-data[dna-publishing guide] specifically for using the MDT.
[cols="1,1,4,1,1",options="header"]
|===
| Field name
| Examples
| Description
| Required
| Placement

// The [.break-all]#TCTA…# is to allow the sequence to be broken at any character, rather than stretching the text cell.
| term:mixs[DNA_sequence]
| [.break-all]#TCTATCCTCAATTATAGGTCATAATTCACCATCAGTAGATTTAGGAATTTTCTCTATTCATATTGCAGGTGTATCATCAATTATAGGATCAATTAATTTTATTGTAACAATTTTAAATATACATACAAAAACTCATTCATTAAACTTTTTACCATTATTTTCATGATCAGTTCTAGTTACAGCAATTCTCCTTTTATTATCATTA#
| The DNA sequence (ASV). Taxonomic interpretation of the sequence depends on the technology and reference library available at the time of publication. Hence, the most objective taxonomic handle is the sequence which can be reinterpreted in the future.
| Required (Highly recommended)
| *Taxonomy* or in separate fasta file (*Seqs.fasta*)

| term:mixs[sop]
| https://www.protocols.io/view/emp-its-illumina-amplicon-protocol-pa7dihn
| Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences. +
 +
A reference to a well documented protocol, e.g. using https://protocols.io[protocols.io]
| Recommended
| *Study*

| term:mixs[target_gene]
| 16S rRNA, 18S rRNA, ITS
| Targeted gene or marker name for marker-based studies
| Highly recommended
| *Study*

| term:mixs[target_subfragment]
| V6, V9, ITS2
| Name of subfragment of a gene or marker important to e.g. identify special regions on marker genes like the hypervariable V6 region of the 16S rRNA gene
| Highly recommended
| *Study*

| term:mixs[pcr_primer_forward]
| GGACTACHVGGGTWTCTAAT
| Forward PCR primer that was used to amplify the sequence of the targeted gene, locus or subfragment.
| Highly recommended
| *Study*

| term:mixs[pcr_primer_reverse]
| GGACTACHVGGGTWTCTAAT
| Reverse PCR primer that was used to amplify the sequence of the targeted gene, locus or subfragment.
| Highly recommended
| *Study*

| term:mixs[pcr_primer_name_forward]
| jgLCO1490
| Name of the forward PCR primer
| Highly recommended
| *Study*

| term:mixs[pcr_primer_name_reverse]
| jgHCO2198
| Name of the reverse PCR primer
| Highly recommended
| *Study*

| term:mixs[pcr_primer_reference]
| https://doi.org/10.1186/1742-9994-10-34
| Reference for the primers
| Highly recommended
| *Study*

| term:mixs[env_broad_scale]
| forest biome [ENVO:01000174]
| *Equivalent to env_biome in MIxS v4* +
In this field, report which major environmental system your sample or specimen came from. The systems identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. were you in the desert or a rainforest?). We recommend using subclasses of ENVO’s biome class: +
http://purl.obolibrary.org/obo/ENVO_00000428
| Recommended (ENVO can be browsed and selected interactively in tool)
| *Samples*

| term:mixs[env_local_scale]
| litter layer [ENVO:01000338]
| *Equivalent to env_feature in MIxS v4* +
In this field, report the entity or entities which are in your sample or specimen´s local vicinity and which you believe have significant causal influences on your sample or specimen. Please use terms that are present in ENVO and which are of smaller spatial grain than your entry for env_broad_scale.
| Recommended (ENVO can be browsed and selected interactively in tool)
| *Samples*

| term:mixs[env_medium]
| soil[ENVO:00001998]
| *Equivalent to env_material in MIxS v4* +
In this field, report which environmental material or materials (pipe separated) immediately surrounded your sample or specimen prior to sampling, using one or more subclasses of ENVO´s environmental material class: +
http://purl.obolibrary.org/obo/ENVO_00010483
| Recommended (ENVO can be browsed and selected interactively in tool)
| *Samples*

| term:mixs[lib_layout]
| Paired
| *Equivalent to lib_const_meth in MIxS v4* +
Specify whether to expect single, paired, or other configuration of reads
| Recommended
| *Samples*

| term:mixs[seq_meth]
| Illumina HiSeq 1500
| Sequencing method/platform used
| Highly recommended
| *Study*

| term:mixs[otu_class_appr]
| "dada2; 1.14.0; ASV"
| Approach/algorithm and clustering level (if relevant) when defining OTUs or ASVs
| Highly recommended
| *Study*

| term:mixs[otu_seq_comp_appr]
| "blastn;2.6.0+;e-value cutoff: 0.001"
| Tool and thresholds used to assign "species-level" names to OTUs or ASVs
| Highly recommended
| *Study*

| term:mixs[otu_db]
| "Genbank nr;221", "UNITE;8.2"
| Reference database (i.e. sequences not generated as part of the current study) used to assigning taxonomy to OTUs or ASVs
| Highly recommended
| *Study*
|===

<<<
